generics/helpers/utils.js

```diff
@@ -202,7 +202,9 @@ function filterLocationIdandCode(dataArray) {
 }
 
 function arrayIdsTobjectIds(ids) {
```

Comment:
@borkarsaish65 can we please have proper function descriptions in utils

Reply:
added